cypress/e2e/awx/resources/workflowVisualizerCRUD.cy.ts

```diff
@@ -141,9 +102,11 @@ describe('Workflow Visualizer', () => {
       cy.get('button[data-cy="workflow-visualizer-toolbar-close"]').click();
       cy.verifyPageTitle(`${workflowJobTemplate.name}`);
     });
-    // FLAKY_06_13_2024
-    it.skip('Adds a new node specifically linked to an already existing node.', function () {
-      cy.visit(`/templates/workflow-job-template/${workflowJobTemplate?.id}/visualizer`);
+    it('Adds a new node specifically linked to an already existing node.', function () {
+      cy.navigateTo('awx', 'templates');
+      cy.filterTableByMultiSelect('name', [workflowJobTemplate.name]);
+      cy.clickTableRowLink('name', workflowJobTemplate.name, { disableFilter: true });
+      cy.get('[data-cy="view-workflow-visualizer"]').click();
```

Comment:
@daphnemaeve I believe @vishsanghishetty suggest to replace this one and use ``getByDataCy` whenever possible.